<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Privatechat</class>
 <widget class="QWidget" name="Privatechat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>637</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>501</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3" colspan="2">
     <widget class="QTextBrowser" name="textBrowser_showprivatemsg">
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_sendmsg">
      <property name="text">
       <string>发送消息</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_wzq">
      <property name="text">
       <string>发起五子棋游戏</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2">
     <widget class="QTextEdit" name="textEdit_private_chat">
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
